src/foam/nanos/controller/AppStyles.js

```diff
@@ -12,6 +12,10 @@ foam.CLASS({
   documentation: 'Generic CSS that can be included into the top level controller of foam app. Implement to foam class to use.',
 
   css: `
+    html {
+      font-size: 14px;
+    }
```

Comment:
```suggestion
    :root{
      font-size: 62.5%; // sets default to 10px so 14px would be 1.4rem
    }
```
```suggestion
    html {
      font-size: 14px;
    }
```

Reply:
whats the reason for setting the default to 10px?

Reply:
Math is easier. That's it. Setting it in percentage makes sure we don't override any user set font scaling

Reply:
got it, thanks! updating